PSReadLine/Render.cs

```diff
@@ -1194,6 +1200,16 @@ private bool LineIsMultiLine()
             return false;
         }
 
+        private bool LineIsMultiLine(string text)
```

Comment:
Line 1193 is `LineIsMultiLine()` (no arguments). It should call this function passing `_buffer`.

Reply:
Passing in `_buffer` would mean calling `_buffer.ToString()` when it's not really necessary. I don't think that's preferable.

Reply:
You could accept a Span to avoid the `ToString` call, right?

Reply:
You cannot get a span/readonlyspan from a `StringBuilder`. I updated the code to use `line.Contains('\n')` and have this method removed.